test/fixtures/dockerfiles/multi-stage-as/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM golang:1.7.3 AS builder
+WORKDIR /go/src/github.com/alexellis/href-counter/
+RUN go get -d -v golang.org/x/net/html
+COPY app.go .
+RUN CGO_ENABLED=0 GOOS=linux go build -a -installsuffix cgo -o app .
+
+FROM alpine:latest AS base
+RUN apk --no-cache add ca-certificates
+WORKDIR /root/
+COPY --from=builder /go/src/github.com/alexellis/href-counter/app .
+CMD ["./app"]
+
+FROM base
+
+FROM base AS extended
```

Comment:
How about another fixture that uses regular image tags as `FROM` args, has `AS` names on all but the last `FROM` statements? Like the public example from the dockerhub docs

Reply:
@adrukh I changed the fixture and flipped the order of the stages. I believe this reflects the common scenario.

I also added the Dockerfile example we got from the customer:
https://snyk.slack.com/archives/CBXS33GJY/p1552396933238600?thread_ts=1552396862.238300&cid=CBXS33GJY